### PR TITLE
Run tests on recent commits to Featuretools

### LINKED ISSUE #65: Run tests on recent commits to Featuretools
Recent commits to Featuretools should trigger a dispatch event to run tests in this repo. See this [PR](https://github.com/FeatureLabs/nlp_primitives/pull/48) for an example.

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Checks for recent commits are scheduled to happen every day at 12 pm EST (Closes #65)

### CODE REVIEW

**Comment by jeff-hernandez** [comment]
We need a repo scoped token to test run this workflow

**Review by rwedge — APPROVED** [review]
LGTM!

File: .github/workflows/scheduler.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+on: push
+  # schedule:
+  #   # At 05:00 PM UTC
+  #   - cron: "0 17 * * *"
```
Comment (rwedge) [inline .github/workflows/scheduler.yml:4]: should this be on: schedule?

Comment (jeff-hernandez) [reply .github/workflows/scheduler.yml:4]: Yeah, but first do we want to test run this workflow? This repo (and the others) will need a repo scoped access token

Comment (rwedge) [reply .github/workflows/scheduler.yml:4]: I added an access token to this repo.  Do we want to wait for a new commit to featuretools to test all the way through?

Already tested the "not recent" scenario

Comment (jeff-hernandez) [reply .github/workflows/scheduler.yml:4]: For testing, I can just change the recent value to make it run through on the next commit

Comment (jeff-hernandez) [reply .github/workflows/scheduler.yml:4]: It ran successfully ([run](https://github.com/alteryx/featuretools-tsfresh-primitives/actions/runs/730344416)), I'll revert the changes to schedule

**Review by rwedge — APPROVED** [review]

**Pull request merged**